syntax = "proto3";

package cancel_example;

service CancelService {
    rpc LongRunningOperation (Request) returns (Response);
}

message Request {
    string request_data = 1;
}

message Response {
    string response_data = 1;
}